### ISSUE TITLE
Add Skript

### ISSUE THREAD

**Issue by aabssmc** [engineer-man/piston#638]
Provide links to different compilers/interpreters that could be used to implement this language, and discuss pros/cons of each.
[https://skriptlang.org](https://skriptlang.org)

**Comment by Jonxslays** [comment]
As far as I can tell this is a plugin built specificall ly for minecraft and not a programming language. Not sure how that's supposed to work with piston.

**Comment by Vrganj** [comment]
There's already https://parser.skunity.com/
I don't think it's worth adding, it's a hassle to maintain and resource intensive

**Comment by Aresiel** [comment]
I do not believe [parser.skunity.com ](https://parser.skunity.com/) is able to in any way interpret, compile or otherwise execute Skript code. 

As I read, it merely parses code:
>  Just to let you know, the Parser can't catch logic errors. So make sure your script works before sharing it. 

I do not believe Skript can run without a Minecraft server either.
